util/abs/provision_pe_xl_nodes.rb

```diff
@@ -0,0 +1,85 @@
+# frozen_string_literal: true
+
+require "./setup/helpers/abs_helper.rb"
+include AbsHelper # rubocop:disable Style/MixinUsage
+
+# TODO: specify as env variable or arg?
+ABS_ID = "slv-473"
+
+# TODO: allow different sizes for each node?
+ABS_SIZE = "c5.2xlarge"
+ABS_VOLUME_SIZE = "80"
+
+ROLES_CORE = %w[master
+                puppet_db
+                compiler_a
+                compiler_b].freeze
+
+ROLES_HA = %w[master_replica
+              puppet_db_replica].freeze
+
+# TODO: specify as env variable or arg?
```

Comment:
Currently just using ARGV[0] since this is the only input; created SLV-499 based on our discussion of the best way to provide host info to the script and integrate this with Beaker to run GPLT with the provisioned hosts.